tests/test_env.py

```diff
@@ -74,73 +75,6 @@ def test_venv_pyton_env_empty_string(tmpdir):
     assert e.value.args[0] == "The env_path cannot be an empty string."
 
 
-@pytest.mark.slowtest
-def test_basic_install(tmpdir):
-    """If this test fails, try running "pip uninstall lorem dummy-yummy iplib" before running it."""
-    env_dir1 = tmpdir.mkdir("env1").strpath
-
-    with pytest.raises(ImportError):
-        import lorem  # NOQA
-
-    venv1 = env.VirtualEnv(env_dir1)
-
-    venv1.use_virtual_env()
-    venv1.install_from_list(["lorem"])
-    import lorem  # NOQA
-
-    lorem.sentence()
-
-    with pytest.raises(ImportError):
-        import yummy  # NOQA
-
-    venv1 = env.VirtualEnv(env_dir1)
-
-    venv1.use_virtual_env()
-    venv1.install_from_list(["dummy-yummy"])
-    import yummy  # NOQA
-
-    with pytest.raises(ImportError):
-        import iplib  # NOQA
-
-    venv1 = env.VirtualEnv(env_dir1)
-
-    venv1.use_virtual_env()
-    try:
-        venv1.install_from_list(
-            ["lorem == 0.1.1", "dummy-yummy", "iplib@git+https://github.com/bartv/python3-iplib", "lorem", "iplib >=0.0.1"]
-        )
-    except CalledProcessError as ep:
-        print(ep.stdout)
-        raise
-    import iplib  # NOQA
-
-
-@pytest.mark.slowtest
-def test_install_fails(tmpdir, caplog, monkeypatch):
-    venv = env.VirtualEnv(tmpdir)
-    venv.use_virtual_env()
-    caplog.clear()
-    caplog.set_level(logging.INFO)
-    package_name = "non-existing-pkg-inmanta"
-
-    # monkeypatch pip install to set --no-index for security reasons (anyone could publish this package to PyPi)
-    compose = env.PipCommandBuilder.compose_install_command
-
-    def mock_compose(*args, **kwargs):
-        if "index_urls" in kwargs or len(args) < 5:
-            return compose(*args, **{**kwargs, "index_urls": []})
-        else:
-            return compose(*args[:3], [], *args[4:], **kwargs)
-
-    monkeypatch.setattr(env.PipCommandBuilder, "compose_install_command", mock_compose)
-
-    with pytest.raises(Exception):
-        venv.install_from_list([package_name])
-
-    log_sequence = LogSequence(caplog)
-    log_sequence.contains("inmanta.env", logging.INFO, f"requirements:\n{package_name}")
```

Comment:
This logging no longer happens as we flatten all requirements to to command line now, so this is no longer required I believe